Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -526,6 +526,15 @@ public async Task DeleteById(int id)
                     var competeParticipant = participants.SingleOrDefault(p => p.IsOfficial);
                     var practiceParticipant = participants.SingleOrDefault(p => !p.IsOfficial);
 
+                    var userId = this.userProviderService.GetCurrentUser().Id;
+
+                    if (competeParticipant?.UserId == userId ||
+                        practiceParticipant?.UserId == userId)
```

Comment:
I don't think you need this check.. the participants should already be filtered only for the current user.

Reply:
Done.